src/renderer/store/modules/profiles.js

```diff
@@ -89,6 +89,33 @@ const actions = {
     commit('setProfileList', profiles)
   },
 
+  async updateChannelThumbnail({ getters, dispatch }, { channelThumbnailUrl, channelName, channelId }) {
+    const profileList = getters.getProfileList
+    for (let i = 0; i < profileList.length; i++) {
+      const currentProfile = JSON.parse(JSON.stringify(profileList[i]))
```

Comment:
This is a copy of "current profile"
`currentProfileCopy` might not be a good name but it's not the original object
```suggestion
      const currentProfileCopy = JSON.parse(JSON.stringify(profileList[i]))
```